include/exadg/structure/user_interface/boundary_descriptor.h

```diff
@@ -127,6 +127,42 @@ struct BoundaryDescriptor
     AssertThrow(counter == 1,
                 dealii::ExcMessage("Boundary face with non-unique boundary type found."));
   }
+
+  void
+  set_dirichlet_cached_data(
```

Comment:
What about adding some Doxygen comments here?

Reply:
So far, I hesitated because we have the same set-functions in multiple boundary descriptors (which are different for the different modules in ExaDG).

I am also not sure if this is the final design. Actually, these functions should not be public, but only accessible for the respective `MyModule::SpatialOperator` class, which sets up the data structures for cached-type boundary conditions. Maybe this internal use is why I considered documentation not that important for this function. Another solution could be to not pass this data structure back to the `BoundaryDescriptor` (via a shared pointer), but only to the implementation of operators and kernels, with the need extend the interfaces of such classes (!).

Lange Rede, kurzer Sinn. I could principally add some explanation, but this text will be copied for multiple boundary descriptors :(